tests/foreman/ui_airgun/test_contentview.py

```diff
@@ -1571,6 +1569,210 @@ def test_positive_remove_cv_version_from_default_env(session, module_org):
         assert ENVIRONMENT not in cvv['Environments']
 
 
+@tier2
+def test_positive_remove_promoted_cv_version_from_default_env(session, module_org):
+    """Remove promoted content view version from Library environment
+
+    :id: a8649444-b063-4fb4-b932-a3fae7d4021d
+
+    :Steps:
+
+        1. Create a content view
+        2. Add a puppet module(s) to the content view
+        3. Publish the content view
+        4. Promote the content view version from Library -> DEV
+        5. remove the content view version from Library environment
+
+    :expectedresults:
+
+        1. Content view version exist only in DEV and not in Library
+        2. The puppet module(s) exists in content view version
+
+    :CaseLevel: Integration
+    """
+    puppet_module_name = 'generic_1'
+    author = 'robottelo'
```

Comment:
Any Specific reason are we keeping the author and puppet module_name static?

Reply:
because they are constant for that specific repo. It was created by `robottelo` user and has package with name `generic` in its list

Reply:
OK